res/style.css

```diff
@@ -776,6 +776,8 @@ body, #root, .profileViewer {
 
 .react-contextmenu.react-contextmenu--visible {
   display: block;
+  max-height: 450px;
```

Comment:
Just wondering if this shouldn't be in `%` `vh` so that it can adjust with the size of the window. I'd rather use percentages (it works given where it is in the DOM) because I know `vh` can have some performance issues regarding reflows. For example `75%` seem to work quite fine for me. What do you think?

Reply:
When you zoom with a in % defined height, you get half cut off text and can't optimize for that.
![screen shot 2018-05-15 at 6 27 08 pm](https://user-images.githubusercontent.com/562354/40070472-7509deb0-586e-11e8-8535-4b8e5a62bb9b.png)

This is why I'd rather stick with px values right now. What do you think?

Reply:
Why not go with a max-height of 100vh? It might look a little overwhelming but it won't artificially limit how much information we display, and we don't need to worry about clipping of items partially because it'll be entirely outside of our control.

On the other hand, showing partially clipped items shows as a good reminder that there's we're not showing everything and that the user might want to try scrolling.

Reply:
@julienw The only performance issues with `vh` that I'm aware of are:

 1. You'll reflow when the window size changes. (This is totally expected, and would be the same with percentage values.)
 2. In Firefox for Android, `vh` sizes are affected by the presence of the dynamic toolbar which appears and disappears when scrolling up and down, so you'll reflow and shift whenever you scroll a page up or down. But this isn't relevant for us.

So I think we can feel free to use `vh` units all we like.

Reply:
yep, let's use vh :) I was thinking 100vh was too much and that's why I suggested 75% (or rather 75vh) but really I don't mind, so @zoepage please pick something that you like (but in vh).

(note that with px you'll have the same issue when zooming :) )

Reply:
> (note that with px you'll have the same issue when zooming :) )

If you mean the cut off of the last item, this should not happen with px values, no.

Reply:
oh I see what you meant now. What I meant is that with px the menu could still be taller than the page. (so 2 different things, sorry for misunderstanding you)